content/providers/01-ai-sdk-providers/12-black-forest-labs.mdx

```diff
@@ -0,0 +1,147 @@
+---
+title: Black Forest Labs
+description: Learn how to use Black Forest Labs models with the AI SDK.
+---
+
+# Black Forest Labs Provider
+
+[Black Forest Labs](https://bfl.ai/) provides a generative image platform for developers with FLUX-based models. Their platform offers fast, high quality, and in-context image generation and editing with precise and coherent results.
+
+## Setup
+
+The Black Forest Labs provider is available via the `@ai-sdk/black-forest-labs` module. You can install it with
+
+<Tabs items={['pnpm', 'npm', 'yarn', 'bun']}>
+  <Tab>
+    <Snippet text="pnpm add @ai-sdk/black-forest-labs" dark />
+  </Tab>
+  <Tab>
+    <Snippet text="npm install @ai-sdk/black-forest-labs" dark />
+  </Tab>
+  <Tab>
+    <Snippet text="yarn add @ai-sdk/black-forest-labs" dark />
+  </Tab>
+
+  <Tab>
+    <Snippet text="bun add @ai-sdk/black-forest-labs" dark />
+  </Tab>
+</Tabs>
+
+## Provider Instance
+
+You can import the default provider instance `blackForestLabs` from `@ai-sdk/black-forest-labs`:
+
+```ts
+import { blackForestLabs } from '@ai-sdk/black-forest-labs';
+```
+
+If you need a customized setup, you can import `createBlackForestLabs` and create a provider instance with your settings:
+
+```ts
+import { createBlackForestLabs } from '@ai-sdk/black-forest-labs';
+
+const blackForestLabs = createBlackForestLabs({
+  apiKey: 'your-api-key', // optional, defaults to BFL_API_KEY environment variable
```

Comment:
just for the record: `BFL_API_KEY` is the same API key name that Black Forest Labs use as well ([docs](https://docs.bfl.ai/quick_start/create_account#create-api-account:~:text=Set%20up%20your%20environment))